.gitignore

```diff
@@ -105,3 +105,6 @@ UNKNOWN.egg-info/
 
 # MonkeyType
 monkeytype.sqlite3
+
+# Integration test
+mirrored-files
```

Comment:
I did notice this the other day too. We should get test_runner.py to clean this up with an atexit probably. For another PR.